src/containers/Secrets/Secrets.js

```diff
@@ -502,8 +501,7 @@ function mapStateToProps(state, props) {
   const namespace = namespaceParam || getSelectedNamespace(state);
 
   return {
-    errorMessage:
-      getDeleteSecretsErrorMessage(state) || getSecretsErrorMessage(state),
+    errorMessage: getDeleteSecretsErrorMessage(state),
```

Comment:
do we end up missing some classes of error now?

ideally we'd just have one 'error' state, and set it with the correct details depending on whether it's create, delete, etc.

Reply:
Yeah that's my concern too so just going through it again now (it seemed a little easy, I do think moving towards a (I think you said `global`) error state would thwart these kind of problems from repeatedly occurring)

Reply:
Had a good try creating secrets, switching namespaces etc, even deleting ones (shouldn't be impacted) with errors (e.g. deleting first with a kubectl command), and patching to one we just deleted - that reports a success but I'm wondering if it did anyway on master so will check